README.md

```diff
@@ -6,6 +6,12 @@ Website of the Czech PyLadies chapter / Web českých PyLadies.
 
 *Pokud je kontrolka zelená, povedlo se dostat poslední změny z větve `master` do ostrého provozu. Má-li jinou barvu, stala se [někde po cestě](https://travis-ci.org/PyLadiesCZ/pyladies.cz) chyba.*
 
+
+## License
```

Comment:
ale samotna licence je v anglictine ne? tak to potom davalo smysl mit to hned na zacatku readme a v en